vendor/ddcloud/resource_additional_nics.go

```diff
@@ -0,0 +1,75 @@
+package ddcloud
+
+import "github.com/hashicorp/terraform/helper/schema"
+
+const (
+	resourceKeyNicServerID     = "serverid"
+	resourceKeyNicID           = "id"
+	resourceKeyNicVLANID       = "vlan_id"
+	resourceKeyNicPriavateIPV4 = "private_ipv4"
+	resourceKeyNicPrivateIPV6  = "private_ipv6"
+	resourceKeyIsShutdownOk    = "shutdown_ok"
```

Comment:
What do you think of maybe making this a property on the provider configuration instead?

So something like:

```hcl
provider "ddcloud" {
  region = "AU"
  allow_server_restarts = true
}
```